### ISSUE TITLE
Launch/OpenUrl API is broken on Windows with 7.1+

### ISSUE THREAD

**Issue by ivankravets** [pallets/click#1501]
### Expected Behavior

`click.launch()` API is broken with 7.1 on Windows. We are going to have a quick release of @PlatformIO and lock Click to <7.1. Something is going wrong with Click last time.

```
>>> import click
>>> click.__version___
>>> 7.1.1
>>> click.launch("https://github.com?click_7_1=is_broken")
>>> The system cannot find the file 'https://github.com?click_7_1=is_broken'
>>> 1
```

### Actual Behavior

Everything works on Click < 7.1. See changes:

- https://github.com/pallets/click/blob/7.0/click/_termui_impl.py#L503
- https://github.com/pallets/click/blob/master/src/click/_termui_impl.py#L560

### Environment

* Python version: No matter
* Click version: 7.1+

**Comment by ellbosch** [comment]
Looks like 7.1 broke mssql-cli for us on Windows.

We have a thread tracking the bug [here](mssql-cli broken on latest Windows update--'The system cannot find the file specified.' #437).

**Comment by davidism** [comment]
Fixed by #1543
